src/en/charms-ha.md

```diff
@@ -1,71 +1,77 @@
+maximising your application's availability. 
+
+Each deployed application is considered a 'distribution group'. Every time a
+unit is added to a distribution group, Juju will spread out units to best
+ensure high availability. As long as the charm and the charm's application are
+well written, you can rest assured that IaaS downtime will not affect your
+application.
+
+!!! Note: see [High Availability](./controllers-ha.html) for details on
+enabling high availability for the controller.
 
 Commands you already use for scaling now ensure your applications are always
 available. e.g.
 
 ```bash
 juju deploy -n 10 <application>
 ```
-
-The way this works depends on whether Juju uses Availability Zones or
-Availability Sets for that provider.
-
+The way this works depends on whether Juju uses 'availability zones' or
+'availability sets' for that provider.
 
 ## Availability Zones
 
-Juju supports Availability Zones on Amazon's EC2 and OpenStack-based clouds.
-Openstack Havana and newer is supported, which includes HP Cloud. Older
-versions of OpenStack are not supported. See the per-provider [Install &
-Configure](./getting-started.html) section for more on these and any other
-provider-specific settings.
+Availability zones allow for the automatic and uniform distribution of units
+across a region. A new instance, for example, will be allocated the zone
+with the fewest members of its distribution group.
+
+Juju supports such zones on Google Compute Engine, VMware vSphere, Amazon's
+EC2, OpenStack-based clouds (Havana or newer) and
+[MAAS](http://maas.io/docs/installconfig-zones). See the
+[Clouds](./clouds.html) section of the documentation for more details on these
+and other provider-specific settings.
 
 If you do not specify a zone explicitly, Juju will automatically and uniformly
-distribute units across the available zones within the region. The spread is
-based on density of instance "distribution groups". This can be overridden
-with a placement directive:
+distribute units across the available zones within the region. This can be
+overridden with a placement directive:
 
 ```bash
 juju bootstrap --to zone=us-east-1b
 juju add-machine zone=us-east-1c
 ```
-
-
 ## Azure Availability Sets
 
-Juju supports Availability Sets on Microsoft's Azure.  As long as at least two
+Juju supports availability sets on Microsoft's Azure.  As long as at least two
 units are deployed, Azure guarantees 99.95% availability of the application
 overall.  Exposed ports are automatically load-balanced across all units within
 the application.  Using availability sets disables manual placement and the
 "add-machine" command.
 
-In Juju 1.20 and later, new Azure environments use availability sets by
-default. To disable availability sets, the 'availability-sets-enabled' option
-must be set in environments.yaml like so:
+New Azure environments use availability sets by default. To disable
+this behaviour, the 'availability-sets-enabled' option must be set in
```

Comment:
this also confused me the first time i read it. perhaps : 'the availability-sets-enabled' option must be set to 'false' like this:

or something